code/game/mecha/equipment/tools/other_tools.dm

```diff
@@ -85,7 +88,7 @@
 			var/turf/T = get_turf(target)
 			cooldown_timer = world.time + 3 SECONDS
 			log_game("[key_name(chassis.occupant)] used a Gravitational Catapult in ([T.x],[T.y],[T.z])")
-			return 1
+			return
```

Comment:
dangling return, (a return value at the end of a proc that returns nothing) is unneeded.